src/helpers/visibilityClassnames.ts

```diff
@@ -15,6 +15,8 @@ export const displayFrom = (from: Size) => {
       return display('hidden', 'sm:flex')
     case 'md':
       return display('hidden', 'md:flex')
+    case 'md700':
+      return display('hidden', 'md700:flex')
```

Comment:
What's mean `md700`? May we escape from using it?

Reply:
It helps us to remove the discord-twitter buttons on `width=700`, when right and left sides of navbar touch each other, not sure if we can remove it (maybe there's a way to detect intersection between elements and remove one-by-one, but we want to remove whole social-links block, probably)